java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -333,6 +335,17 @@ public static Scalar fromString(String value) {
     return new Scalar(DType.STRING, makeStringScalar(value.getBytes(StandardCharsets.UTF_8), true));
   }
 
+  /**
+   * Creates a scalar of list from a column view.
+   * All the rows in the column view become the elements of the list inside the scalar.
```

Comment:
The Javadoc needs to clarify what happens with ownership of memory here.  Is the Scalar reusing the data from the ColumnView and thus the ColumnView needs to stay valid as long as the Scalar is in use?  Or is the Scalar making a copy and the view is no longer needed when the call completes?

Reply:
Good suggestion, updated.